advanced_functionality/r_kernel/using_r_with_amazon_sagemaker.ipynb

```diff
@@ -0,0 +1,486 @@
+{
```

Comment:
<p>Missing remaining regions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/797/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
What do you mean?

Reply:
Updated to use sagemaker SDK to retrieve the corresponding path from the region instead of hardcoding values.